src/components/RefinementList.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="items"
+      :refine="state.refine"
+      :searchable="searchable"
+      :search-for-items="state.searchForItems"
+      :show-more="showMore"
```

Comment:
Don't think we need to expose it since it's only for our internal view.

Reply:
it depends a bit, if you're making a custom widget that wraps it, you want to do a ternary like we do as well no?

Reply:
Maybe I miss something but I don't see how the implementer cannot have the value, but they will have to provide it to the slot though. You can leave as it is, doesn't matter. It's just that in the other widget when a props is only required for the view we don't pass it back to the slot.

Reply:
Yes, they could pass the prop themselves too indeed. I’ll leave it out